website/docs/r/storage_account.html.markdown

```diff
@@ -101,6 +111,17 @@ The following attributes are exported in addition to the arguments listed above:
 * `secondary_connection_string` - The connection string associated with the secondary location
 * `primary_blob_connection_string` - The connection string associated with the primary blob location
 * `secondary_blob_connection_string` - The connection string associated with the secondary blob location
+* `identity` - An `identity` block as defined below, which contains the Identity information for this Storage Account.
+
+---
+
+`identity` exports the following:
+
+* `principal_id` - The Principal ID for the Service Principal associated with the Identity of this Storage Account.
+
+* `tenant_id` - The Tenant ID for the Service Principal associated with the Identity of this Storage Account.
+
+-> You can access the Principal ID via `${azurerm_storage_account.test.identity.0.principal_id}` and the Tenant ID via `${azurerm_storage_account.test.identity.0.tenant_id}`
```

Comment:
I don't think we need this?

Reply:
I think it is still useful for the user to be able to see an example on how to access the property. However, if you still think it's not required - I can remove it.

Reply:
Personally I think it should be removed but its very minor and not a blocker